recipes/tensorflow-quantum/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "tensorflow-quantum" %}
+{% set version = "0.5.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tensorflow/quantum/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: c6a4a72366334673c38cd06329f17889278db4dba8060beb891ef3c489fb72a8
+
+build:
+  number: 0
+  skip: true   # [ py<36 or win]
+  script:
+    - export TF_NEED_CUDA=0
+    - export TF_CUDA_VERSION=10.1
+    - ./configure.sh
+    - bazel build -c opt --cxxopt="-O3" --cxxopt="-march=native" --cxxopt="-D_GLIBCXX_USE_CXX11_ABI=0" release:build_pip_package
+    - bazel-bin/release/build_pip_package /tmp/tfquantum/
+    - ${PYTHON} -m pip install {{ name|lower }}-{{ version }}-py3-none-any.whl -vv --no-deps --no-index
+    - ./scripts/test_all.sh
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - bazel
+  host:
+    - future >=0.17.1
+    - mock
+    - numpy 1.19.*
```

Comment:
Is this a requirement of the package or to avoid solving conflicts with Tensorflow? The latter should be fixed with 2.7.0

Reply:
It is defined by upstream: https://github.com/tensorflow/quantum/blob/05f51c7cc21fdbdd0290093240d6ac4c9ad66352/requirements.txt#L4

I'll try to figure out if this is really necessary.

Reply:
Ref https://github.com/tensorflow/quantum/issues/648